classes/simplemap.php

```diff
@@ -516,7 +516,7 @@ function add_distance_field( $radius_value, $units ) {
 			$distance_input .= '</select>';
 
 			return array(
-				'label' => apply_filters( 'sm-search-label-distance', __( 'Select a distance: ', 'simplemap' ), $post ),
+				'label' => apply_filters( 'sm-search-label-distance', __( 'Select a distance: ', 'SimpleMap' ), $post ),
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->